app/templates/reminders/reminder/edit.hbs

```diff
@@ -0,0 +1 @@
+{{ edit-reminder model=model title=model.title }}
```

Comment:
Is there a reason you are passing in the title here instead of calling `model.title` in the component?